gitclass.html

```diff
@@ -11,5 +11,8 @@
 <body>
    <h2>MY NAME IS AYOOLUWA</h2> 
    <h1>AM A SOFTWARE DEVELOPER</h1>
+   
+   <h3>My Name is jackieriel</h3>
```

Comment:
well jack, although you have done a great job. I want you to change the h3 to h2

Reply:
Ok am working on that

Reply:
Am done with the changes